src/Stan/Inspection.hs

```diff
@@ -45,3 +47,21 @@ data Severity
 -- | Show 'Inspection' in a human-friendly format.
 prettyShowInspection :: Inspection -> Text
 prettyShowInspection = show
+
+{- | List of all inspections.
+-}
+inspections :: [Inspection]
+inspections =
+    [ Inspection
+        { inspectionId = Id "HEAD"  -- TODO: we need to come up with naming scheme for inspection ids
```

Comment:
Yes, another good question..
Should we use some coding scheme, like `STAN-0001-HEAD`, so we may have some other inspections in the future (like `GHC-Warning-0001`)? I am really not sure if this is helpful :thinking:

Reply:
I like the `STAN-0001-HEAD` scheme :heart_eyes: I agree with your reasoning that we probably will have other types of warnings, so adding `STAN` as a prefix is a good idea :+1: